### ISSUE TITLE
Clangd crashes after update to version 0.1.32: Server fails to initialize and enters a restart loop

### ISSUE THREAD

**Issue by BhavyaVM1** [clangd/vscode-clangd#735]
**Bug Description**

After updating to version 0.1.32 of the clangd extension, the server fails to initialize and enters a restart loop. This occurs consistently, and the following errors are logged in the output panel.

**Error Logs**

```
[Error - 11:32:54 AM] Client Clang Language Server: connection to server is erroring. Shutting down server.
[Error - 11:32:54 AM] Stopping server failed
Error: Client is not running and can't be stopped. It's current state is: starting
    at oc.shutdown (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:39:8633)
    at oc.stop (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:39:8212)
    at oc.stop (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:39:51156)
    at oc.handleConnectionError (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:39:13298)
    at errorHandler (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:39:11963)
    at /users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:39:21662
    at y1.invoke (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:2:15350)
    at Xn.fire (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:2:16144)
    at writeErrorHandler (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:3:7088)
    at y1.invoke (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:2:15350)
    at Xn.fire (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:2:16144)
    at qs.fireError (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:3:430)
    at Socket.<anonymous> (/users/.vscode-server/extensions/llvm-vs-code-extensions.vscode-clangd-0.1.32/out/bundle.js:3:1263)
    at Socket.emit (node:events:519:28)
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at emitErrorCloseNT (node:internal/streams/destroy:128:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
[Info  - 11:32:54 AM] Connection to server got closed. Server will restart.
[Error - 11:32:54 AM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 11:32:54 AM] Clang Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Info  - 11:32:54 AM] Connection to server got closed. Server will restart.
[Error - 11:32:54 AM] Server initialization failed.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 11:32:54 AM] Clang Language Server client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 11:32:54 AM] Client Clang Language Server: connection to server is erroring. Shutting down server.
[Error - 11:32:54 AM] Stopping server failed
Error: Client is not running and can't be stopped. It's current state is: startFailed
```

**Steps to Reproduce**

> Install or update to clangd extension version 0.1.32.
> Open a C++ project in VS Code with clangd enabled.
> Observe that the clangd server fails to start and logs errors as shown above.

**Expected Behavior**

> The clangd server should initialize successfully without entering a restart loop.

**VS Code Version: 1.95.3 (Universal)
Clangd Extension Version: 0.1.32
OS: Linux.**

**Comment by HighCommander4** [comment]
See #734; should be fixed in version 0.1.33.

**Comment by HighCommander4** [comment]
Duplicate of #734
